src/dconv_wrapper.cpp

```diff
@@ -0,0 +1,23 @@
+#include "dconv_wrapper.h"
+#include "double-conversion.hpp"
+
+#include <Python.h>
+#include <pymath.h>
+
+namespace double_conversion {
+    extern "C" {
+    int dconv_d2s(double value, char *buf, int buflen, int *strlength) {
+        static auto d2s = DoubleToStringConverter(DoubleToStringConverter::NO_FLAGS, "Infinity", "NaN", 'e', -324, 308, 0, 0);
```

Comment:
static DoubleToStringConverter d2s(DoubleToStringConverter::NO_FLAGS, "Infinity", "NaN", 'e', -324, 308, 0, 0);
```suggestion
        static DoubleToStringConverter d2s(DoubleToStringConverter::NO_FLAGS, "Infinity", "NaN", 'e', -324, 308, 0, 0);
```
This fixes compile error